addons/csi-aws-ebs/aws-csi.yaml

```diff
@@ -430,7 +460,7 @@ spec:
         runAsUser: 0
       containers:
         - name: ebs-plugin
-          image: "{{ .InternalImages.Get "AwsEbsCSI" }}"
+          image: {{ .InternalImages.Get "..." }}
```

Comment:
We need to put actual values here (and in other similar places in this file). We should just leave the value that was there before.